buildbot/repack_dmg.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Repacks .dmg volumes as .7z archives
+
+INPUT=$1
+OUTPUT=$2
+TMP_DIR=/tmp/$TMP_PREFIX/$(basename $INPUT).dir
```

Comment:
What's $TMP_PREFIX?

I would recommend instead having the script create a temp dir through the standard mktemp + trap cleanup pattern.

Reply:
Was not aware of those. Thanks for the heads up!